app/models/manageiq/providers/amazon/cloud_manager.rb

```diff
@@ -54,6 +72,14 @@ def ensure_managers
     network_manager.provider_region = provider_region
   end
 
+  def ensure_block_storage_managers
+    build_block_storage_manager unless block_storage_manager
```

Comment:
@Ladas I've merged the two into `ensure_managers` as there is currently no need to have them separated. I also fixed the other rubocop issues.